app/src/common/shared/org/mozilla/vrbrowser/browser/content/TrackingProtectionStore.java

```diff
@@ -112,18 +131,10 @@ public void onInserted(int position, int count) {
                 mListeners.forEach(listener -> listener.onExcludedTrackingProtectionChange(
                         UrlUtils.getHost(exception.uri),
                         true));
+                List<Session> sessions = SessionStore.get().getSessionsByHost(UrlUtils.getHost(permission.url));
+                sessions.forEach(session -> mContentBlockingController.addException(session.getGeckoSession()));
```

Comment:
Why is this necessary? If we are inserting it into the model, hasn't it already been added by a session?

Reply:
Yes, that was unnecessary. Fixed